DESCRIPTION

```diff
@@ -76,6 +76,6 @@ Config/testthat/edition: 3
 Encoding: UTF-8
 Language: en-US
 LazyData: true
-Roxygen: list(markdown = TRUE, roclets = c("collate", "namespace",
+Roxygen: list(markdown = TRUE, roclets = c("rd", "collate", "namespace",
```

Comment:
```suggestion
Roxygen: list(markdown = TRUE, roclets = c("collate", "namespace",
```
The `rd` roclet shouldn't be used as it was replaced by `admiraldev::rdx_roclet`.